getting-started/data-node-authentication.md

```diff
@@ -0,0 +1,89 @@
+# Data Node Authentication [](distdb-auth)
+
+In a multi-node environment, it is necessary to ensure that the access
+node have the necessary roles and permissions on all the data nodes
+where distributed hypertables are placed.
+
+This is critical since when TimescaleDB is executing a query or
+command it will use the same role on the data nodes as on the access
+node. This means that the same roles need to exist on the data nodes
+and the access node need to have authentication set up for those
+roles.
+
+In a typical setup, you will create the distributed hypertable in a
+database and use the same database on all the data nodes. This means
+that you need to ensure that a user that is connected to the access
+node can access the same databases on the data nodes.
+
+> NOTE: Setting up a secure system is a complex task and this section
+> should not be read as recommending any particular security measures
+> for securing your system. The chapter contains technical
+> instructions for how to set up the different authentication methods.
+
+## Setting up Password Authentication [](distdb-auth.pass)
+
+The easiest authentication to set up is password authentication. It is
```

Comment:
I think this lacks some initial context. I would start this section with something like:

`
An access node needs to connect to and authenticate with data nodes on behalf of the clients it serves. The easiest authentication mechanism to configure is password authentication. Configuring password authentication between the access node and data nodes is a two step process: (1) each data node needs to be configured to accept and authenticate connections using passwords, and (2) the access node needs a list of passwords to use for each user (role) that executes queries and commands across data nodes. We will deal with each step separately below.
`

Reply:
IMHO, there is context earlier in the chapter, but outlining the steps before describing them in detail can be added.

Reply:
Done.